lib/include/ert/enkf/config_keys.h

```diff
@@ -102,8 +102,8 @@ extern "C" {
 #define  LOCAL_CONFIG_KEY                  "LOCAL_CONFIG"
 #define  LOG_FILE_KEY                      "LOG_FILE"
 #define  LOG_LEVEL_KEY                     "LOG_LEVEL"
-#define  LSF_QUEUE_KEY                     "LSF_QUEUE"
-#define  LSF_RESOURCES_KEY                 "LSF_RESOURCES"
+#define  LSF_QUEUE_KEY                     "LSF_QUEUE" // The queue name, e.g. "mr"
```

Comment:
Sorry - I suggest these keys are left unmodified; maybe with a deprecation warning. The keys which are actually accessed through the dictionary based api - and hence everest are [these keys](https://github.com/Statoil/libres/blob/master/lib/include/ert/job_queue/lsf_driver.h#L32).

Reply:
I think that's done now?  Can you check and see if this is satisfactory?  (Perhaps I've misunderstood something, though.)